src/bpftrace.cpp

```diff
@@ -1673,6 +1680,52 @@ std::string BPFtrace::resolve_uid(uintptr_t addr) const
   return username;
 }
 
+std::string BPFtrace::resolve_timestamp(uint32_t strftime_id,
+                                        uint64_t nsecs_since_boot)
+{
+  auto fmt = strftime_args_[strftime_id].c_str();
+  char timestr[STRING_SIZE];
+  struct tm tmp;
+  std::ifstream file("/proc/stat");
+  if (!file)
+  {
+    std::cerr << "Fail to open file /proc/stat: " << std::strerror(errno)
+              << std::endl;
+    return "";
```

Comment:
I'm wondering if it'd be more helpful to return `"(?)"`. Any thoughts?

Reply:
Yeah, totally agree. It could be confusing from users' side to see nothing printed out.